website/blog/2021-02-08-64updates.md

```diff
@@ -0,0 +1,170 @@
+is_blog: true
+published_date: February 8, 2021
+---
+
+React Native 0.64 is out! Which means it's time to showcase all the work we've put in around supporting more community modules, TurboModules, APIs improvements, and even some tools, all in the effort to improve the developer experience.
+
+<!--truncate-->
+
+There's been a lot of work on improving the developer experience, as that's been a focus for us as a whole across all our frameworks and platforms, and React Native for Windows is no exception! Through tooling, dogfooding our own work, and "under the hood" improvements, we're looking forward to sharing with you some of the highlights in this release that we're especially proud to announce.
+
+# 📣 React Native Gallery App
+As part of our growing effort to improve the APIs, developer experience, and end-to-end story of developing React Native apps for Windows, we've decided to build and ship our own app, called the [React Native Gallery](https://github.com/microsoft/react-native-gallery).
+
+The app's main purpose is to showcase all the current functionality available in React Native for Windows, while also being a way for us to test cross-platform community modules and keep ourselves honest about the process of pushing an app to the [Windows Store](https://www.microsoft.com/store/apps/windows).
+
+![rn-gallery-gif](assets/gallery_demo_1.gif)
+
+Much like the [XAML Controls Gallery](https://www.microsoft.com/p/xaml-controls-gallery/9msvh128x2zt) for WinUI, we intend to have the React Native Gallery be available not only as a proving ground for our own work, but also a place where native and JavaScript components alike can be seen in action in the Windows ecosystem. This includes providing lightweight code snippets on each page for developers to copy out and use in their own scenarios.
+
+## 🧪 Try it out now!
+To get your hands on a copy of the app while we wait for final Store approval, head on over to the repo and check out the [certified install on our 0.64 branch](https://github.com/microsoft/react-native-gallery/tree/v0.64.0-preview.5/windows/rngallery_1.0.2.0_Win32_Test).
+
+Alternately, you can get the latest state of the app and the source to test things out directly on our [main repo branch](https://github.com/microsoft/react-native-gallery).
+
+# ✨ Developer Goodness
+As with many of our releases, there's always a number of developer and "under the covers" upgrades we make to the framework as a whole. These usually includes things like performance improvements, better API parity and of course general bug fixes, but let's take a look at some of the notable items available this release.
+
+## 🎉 API parity highlights
+
+More component properties supported by the other platforms now work on Windows! Check them out.
+
+### ⚙ `Platform.version`
+The core API `Platform` has now been extended to include Windows. This means you can improve your styles and refine your Windows app experience.

```

Comment:
Platform.OS and Platform.select did work in older versions. Just not Platform.version

There's a code sample for Platform.version in docs you could maybe use. A bit less exciting though.